lib/utils.js

```diff
@@ -44,8 +44,8 @@ const ASYNC_QUERIES_COMBINATIONS = combineQueries(
 );
 
 const ALL_QUERIES_COMBINATIONS = [
-  SYNC_QUERIES_COMBINATIONS,
-  ASYNC_QUERIES_COMBINATIONS,
+  ...SYNC_QUERIES_COMBINATIONS,
+  ...ASYNC_QUERIES_COMBINATIONS,
```

Comment:
I hope this is OK (`ALL_QUERIES_COMBINATIONS ` wasn't used until now)

Reply:
Damn 🤦‍♂️ Thanks for this fix!